src/index.css

```diff
@@ -59,7 +62,7 @@ p {
     float: right;
     height: 2vw;
     box-sizing: border-box;
-    background-color: white;
+    background-color: rgb(203, 231, 198);

```

Comment:
Why have you chosen to use an RGB color here, but hex everywhere else? Keep it consistent.